### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yfarjoun**
mostly whitespace changes....putting sentences on one line

However:
- did you mis-state N and N-1?
- some capitalization
- typos
- rewording the sentence about sort order

### CODE REVIEW

**Comment by jdidion** [comment]
What do you have against paragraphs? :)

**Comment by nh13** [comment]
I'll butt in here: git is line oriented, so if you have a whole paragraph on one line, a single typo will cause git to track the whole paragraph as being changed.  If you have each sentence in their own line, then a single typo will not affect the whole paragraph, but only that sentence.  This is easier for looking back on history and seeing documentation changes, and a reasonable compromise.  Perhaps I just don't read raw markdown and so don't need it to "look good" in markdown format (i.e. paragraphs on the same line), and rely on the rendering?

**Comment by jdidion** [comment]
But GitHub renders separate lines with a newline between them. Or at least that's how it appears when I look at the rendered readme here https://github.com/yfarjoun/stitch/tree/patch-1.

**Comment by yfarjoun** [comment]
my bad....removed the spaces after the end of lines....I hope it's better now...

**Comment by jdidion** [comment]
@yfarjoun I think you're right - should be `N` and `N + 1`

**Review by jdidion — APPROVED** [review]

**Comment by yfarjoun** [comment]
I don't have merge permission here...go ahead!

File: README.md [hunk 1]
```diff
@@ -71,47 +74,42 @@ cargo build --release
...
-The `-p`/`--pre-alignment` option may be used to pre-align the read using banded local alignment
-to select only those reads with a minimum pre-alignment score to perform the full alignment.  Additional
-options are availabe to control the k-mer size, band-width, and minimum alignment score for this step.
-Furthermore, the full alignmnet can be limited to align the read to just those reference contigs
-with minimum alignment score with the `-x`/`--pre-align-subset-contigs`.  This is useful when 
-aligning to a large database of individual constructs as contigs, where the read is expected only
-to align to a small subset of the contigs.
+The `-p`/`--pre-alignment` option may be used to pre-align the read using banded local alignment to select only those reads with a minimum pre-alignment score to perform the full alignment.
+Additional options are availabe to control the k-mer size, band-width, and minimum alignment score for this step.
+Furthermore, the full alignmnet can be limited to align the read to just those reference contigs with minimum alignment score with the `-x`/`--pre-align-subset-contigs`.
```
Comment (jdidion) [inline README.md:97]:
```suggestion
Furthermore, the full alignment can be limited to align the read to just those reference contigs with minimum alignment score with the `-x`/`--pre-align-subset-contigs`.
```

Comment (yfarjoun) [reply README.md]: nice catch. please add this.

**Pull request merged**